### PR TITLE
Fxml mainwindow

### PR DESCRIPTION
**Pull request by alfaloo**

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml
Comment (Alteqa) [inline src/main/resources/view/MainWindow.fxml:1]: I like the new aesthetic update.

File: src/main/resources/view/LightTheme.css
Comment (Alteqa) [inline src/main/resources/view/LightTheme.css:1]: Looks great and well formatted!

**Review by Alteqa — APPROVED** [review]
All looks good to me

**Review by Kappaccinoh — APPROVED** [review]
I'm liking the new white UI - looks clean and suits the hospital vibe more

new ui

**Pull request merged**